R/data-combined.R

```diff
@@ -548,10 +548,10 @@ DataCombined <- R6::R6Class(
       if (!isIncluded(specifiedNames, currentNames)) {
         missingNames <- specifiedNames[!specifiedNames %in% currentNames]
 
-        message(
-          "Following datasets were specified to be grouped but not found:\n",
-          paste0(missingNames, collapse = "\n")
-        )
+        message(messages$printMultipleEntries(
+          header = "Following datasets were specified to be grouped but not found:\n",
```

Comment:
Suggestion. Move \n in the method instead of having to supply it yourself
ALso move the text to the constant.

Reply:
Good point. I have included this in the function itself.